packages/client/src/components/ui/emojis/emojis.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { FC, useState } from 'react';
+import MockServer from '@/mocks/mock-server';
+import userData from '@/mocks/data/user-data.json';
+import { TEmojis } from '@/types/data-types';
+
+import './emojis.scss';
+
+type EmojiProps = {
+    messageId: number;
+};
+
+const emojis = {
+    like: '👍',
+    hmm: '🫤',
+    heart: '❤️',
+    ghost: '👻',
+    fire: '🔥',
+    the_rooms: '🫃',
+};
```

Comment:
На бэкэнде уже есть хранение эмодзи, нужно будет передавать только название.

```suggestion
const emojis = {
    like: 'like',
    hmm: 'hmm',
    heart: 'heart',
    ghost: 'ghost',
    fire: 'fire',
    the_rooms: 'the_doors',
};
```

Reply:
Спасибо, это поправим уже перед мерджем тогда
А то если сейчас эту правку внесу, то код поломается